packages/core/getUserAvailability.ts

```diff
@@ -158,8 +158,8 @@ const _getCurrentSeats = async (
     where: {
       eventTypeId: id,
       startTime: {
-        gte: dateFrom.format(),
-        lte: dateTo.format(),
+        gte: dayjs(dateFrom).toDate(),
+        lte: dayjs(dateTo).toDate(),
       },
```

Comment:
With dateFrom: 2024-06-12T00:01:000Z and dateTo: 2024-06-12T23:59:59:000Z as inputs, I was not able to find startTime: 2024-06-12T06:30:00.000Z in the DB, for the life of me. With this change, it works. All tests run through. Need to find out why!

Reply:
@emrysal any idea why that could be?

Reply:
It's possible that the ISOString isn't fully compatible with the date call. Alternative fixes are possibly `new Date(dateFrom.format())` or `dateFrom.toDate()` as I think it's already a Dayjs type. But hard to tell without knowing the exact formatted value of dateFrom../dateTo.format()